drivers/serial/uart_sam0.c

```diff
@@ -377,6 +377,90 @@ static void uart_sam0_rx_timeout(struct k_work *work)
 
 #endif
 
+static int uart_sam0_configure(struct device *dev,
+								const struct uart_config *new_cfg)
+{
+	int retval;
+	const struct uart_sam0_dev_cfg *const cfg = DEV_CFG(dev);
+	SercomUsart * const usart = cfg->regs;
```

Comment:
We should probably stick with either `*const` or `* const`.

Reply:
I agree. I initially went with `*const` but that ended with checkpatch shouting at me and not passing the test, so I changed it to `* const`. Unfortunately the whole file has inconsistencies in how `SercomUsart *` declarations are formatted, which could be fixed but I wanted to avoid touching lines not related to the functionality I implemented.

Reply:
Alright, it looks like we will need to fix checkpatch first then.

Marking it as resolved (won't fix).